repository/p.json

```diff
@@ -85,6 +85,16 @@
 				}
 			]
 		},
+		{
+			"name": "Project Management",
```

Comment:
The entries are supposed be sorted by "name".

Reply:
i have fixed that now